test/Jsr292/testngSE90.xml

```diff
@@ -30,7 +30,6 @@
 	<test name="jsr292Test">
 		<classes>
 			<class name="com.ibm.j9.jsr292.AdaptorTests"/>
-			<class name="com.ibm.j9.jsr292.LookupInTests"/>
```

Comment:
Is this only invalid for Java 9?  If its invalid everywhere, should it be deleted?

Reply:
This test file `LookupInTests` in Jsr292 folder only applies to Java8 and Java9 lower than b148. There is another `LookupInTests` in OpenJ9_Jsr_292_API folder which has been updated for Openj9. 

As we don't care b148 anymore, once another PR (https://github.com/eclipse/openj9/pull/281) merged into OpenJ9_Jsr_292_API, I think these two project should be merged together.

Reply:
Thanks for the clarification on the planned approach @TianyuZuo!  Is there a overall issue tracking this plan and the follow-on work?

Reply:
Created an issue to tracking this plan. https://github.com/eclipse/openj9/issues/607